server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -353,6 +353,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion ML_INFERENCE_LLAMA_ADDED = def(9_125_0_00);
     public static final TransportVersion SHARD_WRITE_LOAD_IN_CLUSTER_INFO = def(9_126_0_00);
     public static final TransportVersion ESQL_SAMPLE_OPERATOR_STATUS = def(9_127_0_00);
+    public static final TransportVersion RESOLVE_INDEX_MODE_FILTER = def(9_146_0_00);
```

Comment:
Somehow we've got an odd gap in transport version numbers, but I'm pretty sure you need yours to be at the end.